plugins/pulp_docker/plugins/migrations/0003_tag_schema_change.py

```diff
@@ -0,0 +1,20 @@
+from pulp.server.db.connection import get_collection
+
+
+def migrate(*args, **kwargs):
+    """
+    Add schema_version to the tag collection.
+    """
+
+    schema_version_key = 'schema_version'
+    collection = get_collection('units_docker_tag')
+    # drop old index due to unit_keys fields change
+    index_info = collection.index_information()
+    old_index = 'name_1_repo_id_1'
```

Comment:
actually, i think the new index is created when a new unit is added into the collection after the upgrade, but in case we do just upgrade, without any new content sync we might end up removing old index and not having a new one until sync. i need to check this